src/main/resources/db/migration/changes/MenuItemReview.json

```diff
@@ -0,0 +1,74 @@
+{
+    "databaseChangeLog": [
+      {
+        "changeSet": {
+          "id": "MenuItemReview-1",
+          "author": "Nadav Avital",
+          "preConditions": [
+            {
+              "onFail": "MARK_RAN"
+            },
+            {
+              "not": [
+                {
+                  "tableExists": {
+                    "tableName": "MENUITEMREVIEW"
+                  }
+                }
+              ]
+            }
+          ],
+          "changes": [
+            {
+              "createTable": {
+                "columns": [
+                  {
+                    "column": {
+                      "autoIncrement": true,
+                      "constraints": {
+                        "primaryKey": true,
+                        "primaryKeyName": "CONSTRAINT_5"
```

Comment:
I would suggest picking a different number for "primaryKeyName": "CONSTRAINT_#."
"CONSTRAINT_5" is already used in UCSBDates, and according to a question thread on Slack, the database might complain about it. 

slack link: https://ucsb-cs156-s24.slack.com/archives/C070RLBU51Q/p1714347684240029

Otherwise looks good besides Ron's comment.